src/api/mdn.js

```diff
@@ -115,9 +115,10 @@ module.exports = {
         return string.replace(/\$\w+\$/g, replacePlaceholder);
     },
 
-    /**
-     * Given a string, escapes all quotes within it.
-     */
+    /** Escape the given string for HTML inclusion. */
+    escapeHTML: util.htmlEscape,
```

Comment:
Exposing this on the `mdn` namespace is necessary to allow fixing cases where the page title contains `<` or `>`, which confuses the Kuma sanitiser.